tests/test_examples.py

```diff
@@ -419,8 +423,9 @@ def test_example_extra_files(tmp_path, request):
     ),
     reason="Known issue with conda-standalone<=23.10: shortcuts are created but not removed.",
 )
-def test_example_miniforge(tmp_path, request):
-    input_path = _example_path("miniforge")
+@pytest.mark.parametrize("example", ("miniforge", "miniforge-mamba2"))
+def test_example_miniforge(tmp_path, request, monkeypatch, example):
```

Comment:
```suggestion
def test_example_miniforge(tmp_path, request, example):
```

`monkeypatch` is not used anymore. Actually, `request` isn't either, but I think you could add it to the `_run_installer` command. This adds a finalizer that removes the directory in the end. Sorry that I hadn't thought about this before.